paimon-core/src/test/java/org/apache/paimon/operation/KeyValueFileStoreScanTest.java

```diff
@@ -148,20 +148,20 @@ public void testWithKeyFilter() throws Exception {
     }
 
     @Test
-    public void testWithValueFilter() throws Exception {
+    public void testFilterBucket() throws Exception {
         ThreadLocalRandom random = ThreadLocalRandom.current();
         // 0 <= hr <= 999
         List<KeyValue> data = generateData(100, random.nextInt(1000));
-        writeData(data, 0);
+        writeData(data, "0", 0);
```

Comment:
The behavior of `writeData(List<KeyValue> kvs, String partition, int bucket) ` is very strange. The keyvalue's partition is manually modified in it, but still write the old partition into the data file. 

I recommend deleting this function.